extensions/amp-render/1.0/amp-render.js

```diff
@@ -225,7 +215,7 @@ export class AmpRender extends BaseElement {
         this.togglePlaceholder(false);
       },
       'onRefresh': () => {
-        this.togglePlaceholder(true);
+        this.togglePlaceholder(false);
         this.toggleFallback(false);
       },
```

Comment:
Won't these already be toggled off on refresh? Also, would we want to toggle the loading on refresh, possibly in the Preact layer?

Reply:
1. That's right, these will already be toggled off in `onReady` callback, therefore removed `onRefresh`. 
2. Logic to show/hide placeholder & fallback happens on the AMP side, so moving it to the Preact layer will not be trivial

Reply:
2. No, we should not move placeholder/fallback logic to Preact layer. I am more wondering if Preact layer `refresh` callback should also call `onLoading`? (In AMP layer, this would mean retoggling the loading spinner, which I can't recall if 0.1 amp-list does or not, but seems like a reasonable UX behavior)

Reply:
I see what you mean. By default amp-list does not show the loading indicator during refresh, unless `reset-on-refresh` attribute is specified. Currently, we don't support `reset-on-refresh` for amp-render, but it's something we can revisit if we get some feedback on it.

Reply:
Sounds good, thank you!